tlair/src/test/scala/at/forsyte/apalache/tla/lir/IrGenerators.scala

```diff
@@ -76,7 +84,8 @@ trait IrGenerators {
    */
   def genInt: Gen[ValEx] = for {
     i <- arbitrary[Int]
-  } yield ValEx(TlaInt(BigInt(i)))
+    tt <- oneOf(Untyped(), Typed[Int](i))
+  } yield ValEx(TlaInt(BigInt(i))).withType(tt)
```

Comment:
I think we should have different arbitrary generators for the integer value and the type ID, right? Otherwise it seems like this would constraint the tests to a weird invariant.